crates/bevy_asset/src/handle.rs

```diff
@@ -92,15 +93,6 @@ pub struct StrongHandle {
     pub(crate) drop_sender: Sender<DropEvent>,
 }
 
-impl Drop for StrongHandle {
```

Comment:
@cart can you confirm if it's safe removing the Drop impl for StrongHandle, and implementing it on Handle instead?

Reply:
Nvm